source/_posts/2017-05-19-kanban-method-in-a-consulting-company.md

```diff
@@ -0,0 +1,128 @@
+---
+author: Michał Olszewski
+layout: post
+title: "Kanban method in a consulting company"
+date: 2017-05-12 12:00:00 +0100
+comments: true
+categories: Managament 
+---
+
+# Kanban method in a consulting company
+
+
+
+Before we start we need to know what is that Kanban. Let me show you a short
+explanation:
+
+> **Kanban** is a method for managing resources necessary for production process
+so that everyone produces much as need at a given moment. Work items are
+visualized to give participants a view of progress and process, from task
+definition to customer delivery. Team members "pull" work as capacity permits,
+rather than work being "pushed" into the process when requested.
+
+> In software development, Kanban provides a visual process-management system
+which aids decision-making about what, when and how much to produce. Although
+the method (inspired by the Toyota Production System and lean manufacturing
+originated in software development and IT, it may be applied to any professional
+service whose work outcome is intangible rather than physical.
+
+~ [Wikipedia](https://en.wikipedia.org/wiki/Kanban)
+
+## Tools
+
+The basic work tools are boards and stickers, but in our company we use visual
+boards in [Trello](https://trello.com/), which streamlines our work in kanban.
+Our boards are divided into appropriate categories: Backlog, To Do, In Progress,
+Verification, Done and Project Status, where we can discuss project features
+or problems.
+
+![1](http://imageshack.com/a/img922/6156/YfxSE1.png)
+
+## Cards (Kanbans)
+
+According to the Kanban method we give out a kanban cards. Every card must have:
+
+* Title - As needed, it can be very general or low-level,
+* Description - With some documentation, links or to do list,
+* Assigne - If you want to plan something with others You can assing more than
+one person, but we recommend to assing one person / per task,
+* Label - According to the task,
+* Due date - We set a maximum of two weeks to complete the tasks.
+
+The card is very important in the whole project process. Thanks to
+her, we measure our work and set time for each stage in the design, develop
+and testing solutions in embedded systems projects. Additionally inside the
+card we can discuss the features and see, what happened since last time.
+
+![img](http://imageshack.com/a/img924/2586/spqDrv.png)
+
+## Workflow
+
+The card life begins in Backlog, tasks here's have lowest priority and they
+waiting for select to development - but it's not a rule, sometimes we add cards
+with high priority directly to ToDo column. ToDo is the queue for the most
+important column - In Progress. From there our goal is to safely move this task
+to done. Of course sometimes something will happen, but by analyzing the boards

```

Comment:
@molszewski9412 `Of course sometimes something will happen` very vague and not professional.